src/components/v1/shared/V1ProjectHandle.tsx

```diff
@@ -13,12 +13,13 @@ export default function V1ProjectHandle({
   const handle = useHandleForProjectId(projectId)
 
   return (
-    <Link
-      to={`/p/${handle}`}
-      style={{ fontWeight: 500, ...style }}
-      className="text-primary hover-text-action-primary hover-text-decoration-underline"
-    >
-      @{handle}
+    <Link href={`/p/${handle}`}>
+      <a
```

Comment:
**question** won't this render `<a><a>`? Or does `Link` render something different?

Reply:
Yeah so nextjs `Link` actually doesn't like non tagged children. Weirdly though, if you use tags when it is wrapped with trans or t, it doesn't work either.

Reply:
oh yeah i see what;s goin on now. all g, cheers.